code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -307,6 +317,23 @@ GLOBAL_LIST_EMPTY(PDAs)
 //BASIC FUNCTIONS===================================
 
 			if("Refresh")//Refresh, goes to the end of the proc.
+			if ("Toggle_Font")
+				//CODE REVISION 2
+				font_index = (font_index + 1) % 4
+
+				switch(font_index)
+					if (0)
+						font_mode = "font-family:\"VT323\", monospace;letter-spacing:1px;"
+					if (1)
+						font_mode = "font-family:\"Share Tech Mono\", monospace;letter-spacing:0px;"
+					if (2)
+						font_mode = "font-family:\"Orbitron\", monospace;letter-spacing:0px; font-size:15px"
+					if (3)

```

Comment:
Make these defines please

Reply:
Defines?

Reply:
Defines look like this
`#define SS_NO_FIRE 2`
and usually go at the top of the file you are gonna use them in
In your case you would want something like
```
#define FONT_VT  0
#define FONT_MONO 1
#define FONT_ORB 3
#define FONT_OTHER 4
//then in code
switch(font_index)
   if(FONT_VT)
       ... stuff
   if(FONT_MONO)
       ... stuff
... etc
```

makes code more readable, and if someone ever changes the values of each define, it's a lot easier since you only need to change it in one place